### PR TITLE
Unit tests for Popup and Panel

### PR DESCRIPTION
**Pull request by irina060981**
Created BaseHelpClass to use for unit tests
Created tests for Popup and PanelCompact

And several small fixes for code consistency.

### CODE REVIEW

File: src/vue/components/popup.vue [hunk 1]
```diff
@@ -233,7 +233,7 @@ export default {
         return '0px'
       }
 
-      return this.moduleConfig.initialPos.left
+      return `${this.moduleConfig.initialPos.left}px`
```
Comment (balmas) [inline src/vue/components/popup.vue]: do we have a test somewhere that ensures that the settings in moduleConfig are just numeric?

File: src/vue/components/popup.vue [hunk 2]
```diff
@@ -516,11 +516,13 @@ export default {
         .on('resizemove', this.resizeListener)
     }
 
-    const boundsCheck = this.isWithinBounds()
-    if (!boundsCheck.withinBounds) {
-      this.shift.x += boundsCheck.adjX
-      this.shift.y += boundsCheck.adjY
-      this.$options.logger.log(`Popup position has been adjusted to stay within the viewport`)
+    if (typeof this.$el.querySelector === 'function') {
```
Comment (balmas) [inline src/vue/components/popup.vue]: why this test here?

File: src/vue/components/popup.vue (hunk 1 above)
Comment (irina060981) [reply src/vue/components/popup.vue]: I am not sure - it comes as property, so we could add this check inside it loads.
I am not sure, we should place it here.

File: src/vue/components/popup.vue (hunk 2 above)
Comment (irina060981) [reply src/vue/components/popup.vue]: We have similiar check before `interact.js` initialization inside panel.
So I added the same check here for consistency.

**Pull request merged**